e2e/utils.go

```diff
@@ -0,0 +1,33 @@
+package e2e
+
+import (
+	snapapi "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
+	k8sv1 "k8s.io/api/core/v1"
+)
+
+func getPVC(pvcYAML string) (*k8sv1.PersistentVolumeClaim, error) {
+	obj, _, err := deserializer.Decode([]byte(pvcYAML), nil, nil)
+	if err != nil {
+		return nil, err
+	}
+	pvc := obj.(*k8sv1.PersistentVolumeClaim)
```

Comment:
I think `obj` can returned here without casting it to actual resource because the end resource is used in the `crClient.Create` function that can take the object.  
So we can have a single function that just decodes all the input yaml strings to obj.

Reply:
I prefer casting it because it makes the code clearer as to what object it is working on.

Reply:
ok